src/gsy_e/setup/two_sided_market/load_fitlimits_match.py

```diff
@@ -0,0 +1,50 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of Grid Singularity Exchange.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+from gsy_framework.constants_limits import ConstSettings
+from gsy_e.models.area import Area
+from gsy_e.models.strategy.infinite_bus import InfiniteBusStrategy
+from gsy_e.models.strategy.load_hours import LoadHoursStrategy
+
```

Comment:
Can you please add a comment here for what this setup file is used? Ideally it contains the description of the bug. So we know that this is used in the integration tests and do not delete it. Thanks!